terraform/environments/electronic-monitoring-data/dms_data.tf

```diff
@@ -31,10 +31,66 @@ data "aws_iam_policy_document" "dms_target_ep_s3_bucket" {
   }
 }
 
+data "aws_iam_policy_document" "dms_dv_parquet_s3_bucket" {
+  statement {
+    sid = "EnforceTLSv12orHigher"
+    principals {
+      type        = "AWS"
+      identifiers = ["*"]
+    }
+    effect  = "Deny"
+    actions = ["s3:*"]
+    resources = [
+      aws_s3_bucket.dms_dv_parquet_s3_bucket.arn,
+      "${aws_s3_bucket.dms_dv_parquet_s3_bucket.arn}/*"
+    ]
+    condition {
+      test     = "NumericLessThan"
+      variable = "s3:TlsVersion"
+      values   = [1.2]
+    }
+  }
+}
+
 # data "aws_iam_policy_document" "dms_policies" {
 #   statement {
 #     effect    = "Allow"
 #     actions   = ["dms:CreateReplicationSubnetGroup"]
 #     resources = ["*"]
 #   }
 # }
+
+data "aws_iam_policy_document" "dms_dv_glue_assume_role" {
+  statement {
+    actions = ["sts:AssumeRole"]
+
+    principals {
+      identifiers = ["glue.eu-west-2.amazonaws.com"]
+      type        = "Service"
+    }
+  }
+}
+
+data "aws_iam_policy_document" "dms_dv_iam_policy_document" {
+  statement {
+    effect = "Allow"
+    actions = [
+      "s3:GetObject",
+      "s3:ListBucket"
+    ]
+    resources = ["${aws_s3_bucket.dms_dv_glue_job_s3_bucket.arn}/*",
+      aws_s3_bucket.dms_dv_glue_job_s3_bucket.arn,
+      "${aws_s3_bucket.dms_dv_parquet_s3_bucket.arn}/*",
+    aws_s3_bucket.dms_dv_parquet_s3_bucket.arn]
```

Comment:
Also need the dms_target_ep_s3_bucket.arn in here as currently it doesn't have access and this overwrite the S3 policies in the other service roles